src/components/util/memoize.test.js

```diff
@@ -0,0 +1,20 @@
+import memoize from './memoize';
+
+describe('Memoize functional component', () => {
+  it('if it filters', async () => {
+    const fn = (args) => {
+      return args;
+    };
+    const data = await memoize(fn);
```

Comment:
@Mathansivavelmurugesan the `memoize` is not async so I think we can drop the `await` here.